plugin/storage/cassandra/spanstore/operation_names.go

```diff
@@ -25,23 +26,51 @@ import (
 	"github.com/jaegertracing/jaeger/pkg/cache"
 	"github.com/jaegertracing/jaeger/pkg/cassandra"
 	casMetrics "github.com/jaegertracing/jaeger/pkg/cassandra/metrics"
+	"github.com/jaegertracing/jaeger/storage/spanstore"
 )
 
 const (
-	insertOperationName = `INSERT INTO operation_names(service_name, operation_name) VALUES (?, ?)`
-	queryOperationNames = `SELECT operation_name FROM operation_names WHERE service_name = ?`
+	// LatestVersion latest version of operation_names table schema, increase the version if your table schema changes require code change
+	LatestVersion = 1
+	// TableQueryStmt the query statement used to check if a table exists or not
+	TableQueryStmt = "SELECT * from %s limit 1"
```

Comment:
does it need to be public?

Reply:
and the name could be `tableVersionCheckStmt`, since you're not actually querying this table